subprojects/gradle/module-dependencies-shared/src/main/kotlin/com/avito/module/configurations/ConfigurationType.kt

```diff
@@ -2,31 +2,31 @@ package com.avito.module.configurations
 
 import org.gradle.api.artifacts.Configuration
 
-public sealed class ConfigurationType {
+public enum class ConfigurationType {
```

Comment:
Don't see the benefits of using a sealed class here.
Clients have to convert all the time to class.